packages/ui/__stories__/Icon.stories.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import { Clipboard } from 'react-feather'
 
-import { Icon } from '../src/Icon'
+import { Icon } from '../src'
+import { Google } from '@icons-pack/react-simple-icons'
```

Comment:
nit: I would suggest putting absolute imports first

Reply:
Sure, I missed this! Thanks